pkg/apis/crds/karpenter.sh_nodepools.yaml

```diff
@@ -51,37 +51,37 @@ spec:
               Node properties are determined from a combination of provisioner and
               pod scheduling constraints.
             properties:
-              deprovisioning:
+              disruption:
                 default:
+                  consolidateAfter: 15s
                   consolidationPolicy: WhenUnderutilized
-                  consolidationTTL: 15s
-                  expirationTTL: 90d
-                description: Deprovisioning contains the parameters that relate to
-                  Karpenter's deprovisioning logic
+                  expireAfter: 720h
+                description: Disruption contains the parameters that relate to Karpenter's
+                  disruption logic
                 properties:
+                  consolidateAfter:
+                    default: 15s
+                    description: ConsolidateAfter is the duration the controller will
+                      wait before attempting to terminate nodes that are underutilized.
+                      Refer to ConsolidationPolicy for how underutilization is considered.
+                    pattern: ^(([0-9]+(s|m|h))+)|(Never)$
```

Comment:
Calling this out: since duration allows multi-types (like `10m5s`), this will fail for those durations. I think it's an acceptable stance, but a weird gotcha. Any thoughts to allowing this? It would be a tougher regex string for sure.

Reply:
Additionally, I see that the validation logic below just uses the`time.ParseDuration` which allows for multi types, so may be worth aligning here.

Reply:
> this will fail for those durations

It won't fail due to the `+` at the end. That allow you to have 1 or more of the inner group, so you _could_ do something like `10m5s` like you are proposing above.